packages/jest-circus/src/formatNodeAssertErrors.ts

```diff
@@ -3,38 +3,33 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- *
- * @flow strict-local
  */
 
-// $FlowFixMe: Converted to TS. It's also not exported, but should be imported from `matcher-utils`
-import type {DiffOptions} from 'jest-diff';
-import type {Event, State} from 'types/Circus';
+import {DiffOptions} from '@jest/types';
```

Comment:
you can import `DiffOptions` from `jest-matcher-utils`.

I'd do 

```ts
export type DiffOptions = Parameters<typeof jestDiff>[2];
```

in `jest-matcher-utils` until we can export the type properly from `jest-diff`

Potentially, you can do `type DiffOptions = Parameters<typeof diff>[2];` in this file 🙂